docs/pages/admin-guides/management/operations/ca-rotation.mdx

```diff
@@ -138,54 +151,75 @@ also signs client certificates for users connecting to Windows desktops and
 Teleport SSH servers. Teleport-protected servers and Windows desktops use these
 certificates.
 
-Once you have completed the rotation and reached the final `standby` phase,
+Before you complete the rotation and reach the final `standby` phase,
 users who have signed into Teleport must reauthenticate to receive a user
-certificate from the new CA, otherwise Teleport client commands fail.
-
-If you have registered Windows desktops with Teleport, [follow the
-guide](../../../enroll-resources/desktop-access/active-directory.mdx) to export
-the Teleport user CA so the Windows Desktop Service can authenticate to RDP
-hosts. Verify that you can connect to registered desktops throughout the
-rotation.
+certificate from the new CA, otherwise web sessions and Teleport client commands
+will fail and users may have to log out and log back in.
+To avoid this, you can wait longer than the max user session TTL between the
+`update_clients` and `standby` phases so that all user certificates signed by
+the old CA will have already expired and required a re-login.
+
+If you have registered Windows desktops with Teleport,
+[follow the guide](../../../enroll-resources/desktop-access/active-directory.mdx)
+to export the Teleport user CA so the Windows Desktop Service can authenticate
+to RDP hosts.
+Verify that you can connect to registered desktops throughout the rotation.
 
 ### `db` and `db_client`
 
 The `db` and `db_client` CAs issue certificates that the Teleport Database
-Service uses to communicate with self-hosted databases. 
+Service uses to communicate with self-hosted databases.
 
 The Teleport Database Service presents a certificate signed by the `db_client`
 CA when communicating with a self-hosted database, which an admin configures to
-trust certificates issued by the CA. 
+trust certificates issued by this CA.
 
 Admins can configure self-hosted databases to present a certificate signed by
 the `db` CA, which the Database Service uses to verify that a database server is
 a genuine Teleport-protected resource. Alternatively, self-hosted databases can
 present a certificate signed by a custom CA, and admins can configure the
 Teleport Database Service to trust the CA.
 
-#### Beginning the rotation
+#### Rotating the database CAs
```

Comment:
What would you think of merging Step 1 and Step 2, so that:
- the H3-level sections in Step 1 would be the names of CAs (as they are now)
- the content of each section is a list of instructions for rotating the CA

Right now, some H3s in Step 1 are instructions for rotating the CA, and some aren't. Meanwhile, Step 2 is pretty short, and the `tctl status` examples are longer than the instructions themselves.

I can make this change in a PR against this PR if that makes sense to you!

Reply:
I think the current idea is that step 1 explains what each CA is and gives any specific instructions you need to know for whatever is unique about that CA, and then step 2 gives you the CLI commands to actually do a rotation for any CA. I'm not personally sure how i would go about merging them, I wasn't really trying to restructure the guide here, just add some context. If you want to give it a shot in another PR go for it